web_pivot_computed_measure/static/src/pivot/pivot_model.esm.js

```diff
@@ -230,6 +232,8 @@ patch(PivotModel.prototype, {
                     }
                     toEnableFields.push(afield);
                     toEnable.push(toEnableFields);
+                } else if (this.env.debug === "1") {
+                    // You can console log for debugging fieldDef or metaData.fields
```

Comment:
```suggestion
                } 
```
Remove unnecesary code